pegasus/src/database.rb

```diff
@@ -17,17 +17,18 @@ class Tutorials
   # We alias the database columns with names that have the datatype suffixes stripped off for
   # backwards-compatibility with some existing tutorial pages
   # Note: A tutorial can be present in the sheet but hidden by giving it the "do-not-show" tag.
-  def initialize(table)
+  def initialize(table, no_cache=false)
     @table = "cdo_#{table}".to_sym
+
     # create an alias for each column without the datatype suffix (alias "amidala_jarjar_s" as "amidala_jarjar")
-    @column_aliases = CDO.cache.fetch("Tutorials/#{@table}/column_aliases") do
+    @column_aliases = CDO.cache.fetch("Tutorials/#{@table}/column_aliases", force: no_cache) do
```

Comment:
Just a post-merge note that the use of `cache_classes = false` [here](https://github.com/code-dot-org/code-dot-org/blob/c5d2084673d65ad162f1d1342650cba3e091f5ef/dashboard/config/environments/staging.rb#L5) was apparently intended to disable this caching for staging.  Not yet sure why it didn't seem to prevent this caching.